app/assets/stylesheets/index.scss

```diff
@@ -61,3 +78,30 @@ a {
 .pagination {
   text-align: center;
 }
+
+
+.footer {
+  background-color: #292c2f;
+  font-size: 1.2rem;
+  padding: 2rem;
+  text-align: center;}
+  .footer p.slogan {
+    color: #8d9093;
```

Comment:
Color literals like `#8d9093` should only be used in variable declarations; they should be referred to via variable everywhere else.